cypress/integration/inventory/quick-marc.spec.js

```diff
@@ -51,7 +53,6 @@ describe('ui-inventory: MARC', () => {
         cy.expect(Page.has({ url: matching(/\/inventory\/quick-marc\/edit/) }));
       });
 
-      // FIXME: I can't open edit page because of disabled edit button to rewrite by using interactors
       describe('adding a new field', () => {
```

Comment:
I can't open edit page because of disabled edit button to rewrite tests by using interactors

Reply:
@BogdanDenis @ViktorSoroka07 of @JohnC-80 How I can open edit page for the MARC record?